.gitignore

```diff
@@ -242,6 +242,7 @@ pacemaker_*.info
 /cts/scheduler/bug-rh-1097457.trs
 /cts/scheduler/shadow.*
 /cts/test-suite.log
+/lib/*/fuzzers/*/*_fuzzer
```

Comment:
Basically git should work with symlinks. Afaik we're not using them in the pacemaker-repo so far and I don't know if it is worth the possible risks or just the effort for something temporary. But maybe worth thinking about it though.

Reply:
symlinks do work in the repo but not worth it for this